content/posts/2022-10-08-why-good-code-style-matters.md

```diff
@@ -1,8 +1,10 @@
 ---
 title: Why good code style matters
 date: 2022-10-24T10:10:19.885Z
-featuredImage: /post-images/good-style-article-prettier.png
-draft: false
+
+featuredImage:
```

Comment:
@ntorbinskiy It is only for local testing? It should not be in the PR to break things